DependencyInjection/Configuration.php

```diff
@@ -130,11 +130,20 @@ public function getConfigTreeBuilder()
                     ->useAttributeAsKey('id')
                     ->prototype('array')
                         ->fixXmlConfig('context')
+                        ->fixXmlConfig('template')
                         ->fixXmlConfig('setting')
                         ->children()
                             ->arrayNode('contexts')
                                 ->prototype('scalar')->end()
                             ->end()
+                            ->arrayNode('templates')
+                                ->prototype('array')
+                                    ->children()
+                                        ->scalarNode('name')->cannotBeEmpty()->end()
```

Comment:
can we use ``useAttributeAsKey()``here instead of `name` and `template`?

like:
```yml
templates:
    foo: bar.html.twig
```